components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/java/org/wso2/carbon/identity/application/mgt/ui/UiUtil.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.application.mgt.ui;
+
+import javax.servlet.http.HttpServletRequest;
+
+public class UiUtil {
+    static public ApplicationBean getApplicationBeanFromSession(HttpServletRequest httpServletRequest, String spName) {
+        if (httpServletRequest.getSession().getAttribute(spName) == null) {
+            ApplicationBean applicationBean = new ApplicationBean();
+            httpServletRequest.getSession().setAttribute(spName, applicationBean);
+        }
+        return (ApplicationBean)httpServletRequest.getSession().getAttribute(spName);
+    }
+}
```

Comment:
1. Rename the class to ApplicationMgtUIUtil
2. Use httpServletRequest.getSession(false) and don't forget to do null check